extension/landing-zone/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
```

Comment:
Remove this comment, as this is a DA extension, so it will lock into an exact version as all DAs should. I'll push a renovate rule update that will allow this version to be bumped. You might aswell lock it into the latest version now - 1.57.0